web_notify_channel_message/models/mail_channel.py

```diff
@@ -1,5 +1,4 @@
-# pylint: disable=missing-docstring
-# Copyright 2016 ACSONE SA/NV
```

Comment:
You should keep copyright from ACSONE here.

Reply:
Hi Remi, I just deleted this copyright because it was a mistake by copying the copyright by issue at creating, actually ACSONE is not  involved in this module (you can check commit history).

Reply:
Fine, thank you for clarifying !